### ISSUE TITLE
sonata_type_model_autocomplete with mutliple false are broken

### ISSUE THREAD

**Issue by Skunkz** [sonata-project/SonataAdminBundle#3227]
Hi,
Since fews days ago, i've got a js error in select2 with my sonata_type_model_autocomplete, only when multiple false.
ReferenceError: opts is not defined
select2.min.js (line 22, col 20601)
The problem seems to be known and fixed on the select2 side : https://github.com/select2/select2/commit/a7e8f2bfade064245a442c0245453cb0f6fdc57f
Do you it's possible to update the select2 versions used by sonata admin to fix the problem?
Thanks

**Comment by torst** [comment]
please put your code to see how it look

**Comment by Skunkz** [comment]
Hi,
So a basic example that used to work properly : 
```php
 /**
     * Configure the fields to be displayed in the create/edit form.
     *
     * @param FormMapper $formMapper
     */
    protected function configureFormFields(FormMapper $formMapper)
    {
        $formMapper
            ->with('General', ['class' => 'col-md-6'])
                ->add('title')
                ->add('subTitle')
                ->add('type', 'choice', ['label' => 'Type', 'choices' => $this->typeChoices])
                ->add(
                    'category',
                    'sonata_type_model_autocomplete',
                    [
                        'property' => 'label',
                        'multiple' => false
                    ]
                )
                ->add('active', null, ['label' => 'Is Active', 'required' => false])
            ->end();
}
```
The category field is the one causing js problems ( it works but it's not pre populated anymore) 
If I change the bower.json of sonata-project/core-bundle:
from
```php
 fix "dependencies": {
        ...
        "select2": "^3.5.4",
        ...
}
```
to
```php
 fix "dependencies": {
        ...
        "select2": "^3.5",
        ...
}
```
The problem is tfixed.

**Comment by pulzarraider** [comment]
Duplicate of #3206.Fixed with sonata-project/SonataCoreBundle#189. Thank you for reporting the issue.
